pypeit/spectrographs/vlt_sinfoni.py

```diff
@@ -0,0 +1,332 @@
+"""
+Module for Keck/MOSFIRE specific methods.
```

Comment:
Looks like this was copied from keck_mosfire and then modified. All of the Keck/MOSFIRE comments need to be updated.

Reply:
Fixed.